### PR TITLE
Propose PERPUSD and USDPERP price identifiers

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: UMIPs/priceid-perpusd.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    type: "expression",
+    expression: `
+      median( POOL_USDT_BINANCE, POOL_USDT_OKEX, POOL_USD_CBPRO )
+    `,
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    customFeeds: {
+      POOL_USDT_BINANCE: { type: "cryptowatch", exchange: "binance", pair: "perpusdt" },
+      POOL_USDT_OKEX: { type: "cryptowatch", exchange: "okex", pair: "perpusdt" },
+      POOL_USD_CBPRO: { type: "cryptowatch", exchange: "coinbase-pro", pair: "perpusd" },
```
Comment (Reinis-FRP) [inline UMIPs/priceid-perpusd.md:50]: POOL should better be replaced to PERP in the variable naming

File: UMIPs/priceid-perpusd.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+
+## Rationale
+
+The three most voluminous exchanges for the PERP token are Binance, OKEx and Coinbase Pro. Pricing information for all three of these is readily available and suitable for pricing the PERP token.
+
+## Implementation
+
+```
+1. Query the POOL/USD price from Binance, OKEx and Coinbase Pro using whatever TWAP is defined for the ancillary data `twapLength` value. Note that `twapLength` should be defined in seconds. If there is no ancillary data or no `twapLength` key present, then the most recent price that falls before the price request timestamp should be used.
```
Comment (Reinis-FRP) [inline UMIPs/priceid-perpusd.md:92]:
```suggestion
1. Query the PERP/USD price from Binance, OKEx and Coinbase Pro using whatever TWAP is defined for the ancillary data `twapLength` value. Note that `twapLength` should be defined in seconds. If there is no ancillary data or no `twapLength` key present, then the most recent price that falls before the price request timestamp should be used.
```

File: UMIPs/priceid-perpusd.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+2. Take the median of the PERP/USD results from step 1.
+3. Round result from step 2 to 8 decimals to get the PERP/USD price.
+4. (for USD/PERP) Take the inverse of the result of step 3.
+5. (for USD/PERP) Round result from step 6 to 8 decimals to get the USD/PERP price.
```
Comment (Reinis-FRP) [inline UMIPs/priceid-perpusd.md:96]:
```suggestion
5. (for USD/PERP) Round result from step 4 to 8 decimals to get the USD/PERP price.
```

**Comment by smb2796** [comment]
Thanks for catching my oversights @Reinis-FRP. should be good now

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**